src/Resources/config/services/search_engine.xml

```diff
@@ -5,9 +5,9 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="lakion_sylius_elastic_search.search.elastic_engine" class="Lakion\SyliusElasticSearchBundle\Search\Elastic\ElasticSearchEngine">
-            <argument type="service" id="fos_elastica.manager.orm" />
-            <argument type="service" id="lakion_sylius_elastic_search.factory.search" />
+        <service id="sylius_elastic_search.search.elastic_engine" class="Sylius\ElasticSearchPlugin\Search\Elastic\ElasticSearchEngine">
```

Comment:
Do we even need this amount of `elastic` and `search` keywords in the namespace ?